src/pocketmine/MemoryManager.php

```diff
@@ -200,10 +200,7 @@ public function triggerGarbageCollector(){
 		$cycles = gc_collect_cycles();
 		
 		foreach($this->server->getLevels() as $level){
-			if($this->cacheTrigger){
-				$level->clearCache(true);
-			}
```

Comment:
I think removing this is a bad idea. After some time, the cached material is not going to be useful anymore, and should be fully cleared.